### DISCUSSION TITLE
How to require Vercel build not to cache specified packages?

### DISCUSSION THREAD

**Discussion by AH-dark** [vercel/community#5277] [Help]
### Summary

I’m using Turborepo to build a project, where I store article data in one package and use Next.js to build pages in another package. Next.js reads the data at startup through the previously installed package. 

How can I configure it so that Vercel does not cache the package storing the data, preventing the pages from not updating after I commit the code?

### Steps to Reproduce

- Create a package named @example-proj/source.
- Create another package named @example-proj/web.
- Install the @example-proj/source package into @example-proj/web and utilize it.
- Commit the changes and build the project in Vercel.

Following this process, if I update the @example-proj/source package, @example-proj/web will still use the cached version and not update the posts.

**Comment by mehulkar** [comment]
Add a file `@exmaple-proj/source/turbo.json` with the contents:

```json
{
  "extends": ["//"],
  "pipeline": {
    "build": {
      "cache": false
    }  
  }
}
```

This will effectively tell the `source` package not to cache its outputs. This solution is not specific to Vercel, however. It will opt out of caching locally also.
